search-app/docs/api.md

```diff
@@ -2443,7 +2443,7 @@ A granule is downloadable when it contains at least one RelatedURL of type GETDA
 
 A granule is browsable when it contains at least one RelatedURL of type GET RELATED VISUALIZATION.
 
-    curl "%CMR-ENDPOINT%/collections?browsable=true"
+    curl "%CMR-ENDPOINT%/granules?browsable=true&provider=PROV1"
```

Comment:
https://eosdis.slack.com/archives/G6742FUPN/p1682702557878169 From this slack message this should be updated from a different ticket but, was missed.